project-minimization-core/src/main/kotlin/org/plan/research/minimization/core/algorithm/graph/condensation/CondensedEdge.kt

```diff
@@ -0,0 +1,12 @@
+package org.plan.research.minimization.core.algorithm.graph.condensation
+
+import org.plan.research.minimization.core.model.DDItem
+import org.plan.research.minimization.core.model.graph.GraphEdge
+
+data class CondensedEdge<V : DDItem, E : GraphEdge<V>>(
+    val from: CondensedVertex<V>,
+    override val to: CondensedVertex<V>,
+    val originalEdges: List<E>,
```

Comment:
Is it necessary to store it?

Reply:
It could contain some important information such as the nature of the link (what it overrides, character of usage of the method, etc.), and we could use it in the future